grammar.js

```diff
@@ -1589,9 +1585,9 @@ class ThenExpect {
     }
 
     return(scope) {
-        var args = [];
-        for (var i = 0; i < arguments.length; i++) {
-            args.push(arguments[i]);
+        const args = [];
+        for (const arg of arguments) {
+            args.push(arg);
         }
```

Comment:
Follow-up: We have rest args now! This ridiculous optimization is no longer vogue.